src/main/io/serial_cli.c

```diff
@@ -2494,6 +2497,25 @@ static void cliMotor(char *cmdline)
     cliPrintf("motor %d: %d\r\n", motor_index, motor_disarmed[motor_index]);
 }
 
+static void cliName(char *cmdline)
+{
+    
+    uint32_t len = strlen(cmdline);
+    if (isEmpty(cmdline)) {
```

Comment:
In addition there is no way to set the name to empty, perhaps some kind of control e.g. "NONE", "-" or a switch command e.g. -remove.